src/views/Nft/market/components/BuySellModals/SellModal/index.tsx

```diff
@@ -93,7 +93,8 @@ const SellModal: React.FC<SellModalProps> = ({
   const { account } = useWeb3React()
   const { callWithGasPrice } = useCallWithGasPrice()
   const { toastSuccess } = useToast()
-  const collectionContract = useErc721CollectionContract(nftToSell.collectionAddress)
+  const collectionContractReader = useErc721CollectionContract(nftToSell.collectionAddress, false)
+  const collectionContractSigner = useErc721CollectionContract(nftToSell.collectionAddress)
```

Comment:
I think we could do the same with this one

Reply:
Done